atom/browser/native_window_views.cc

```diff
@@ -490,11 +496,15 @@ bool NativeWindowViews::IsMaximizable() {
 #endif
 }
 
-void NativeWindowViews::SetFullScreenable(bool maximizable) {
+void NativeWindowViews::SetFullScreenable(bool fullscreenable) {
+  #if defined(OS_WIN)
+    FlipWindowStyle(GetAcceleratedWidget(), false, WS_MAXIMIZEBOX);
```

Comment:
The maximize/zoom button thing is only limited to OS X, on OS X the zoom button is used as fullscreen button by default.